### ISSUE TITLE
[Bug]: BottomSheet still reachable by screen reader when not visible

### ISSUE THREAD

**Issue by rimabadrandmde** [gorhom/react-native-scroll-bottom-sheet#2385]
### Version

v5

### Reanimated Version

v3

### Gesture Handler Version

v2

### Platforms

iOS, Android

### What happened?

In our project we are showing the BottomSheet with a button in it. On button tap we open a dismissable bottomSheetModal on top of the bottomSheet. Unfortunately the content of the BottomSheet is still being reachable by the screen reader on iOS and Android when it is covered by the BottomSheetModal which is confusing for the users.

### Reproduction steps

- Use our example code
- tap on button in the BottomSheet which triggers opening the BottomSheetModal on top of the BottomSheet
- activate screen reader
- navigate with the screen reader 
—> bottomSheet header and content are still reachable even though the bottomSheet is covered by the BottomSheetModal

### Reproduction sample

https://snack.expo.dev/@rimabadran/bottom-sheet---accessibility-issue

### Relevant log output

```shell

```
